pkg/reconciler/workload/synctarget/synctarget_reconcile.go

```diff
@@ -48,6 +48,10 @@ func (c *Controller) reconcile(ctx context.Context, syncTarget *workloadv1alpha1
 
 	var rootShardKey string
 	for _, workspaceShard := range workspaceShards {
+		// Don't include shards whose sole purpose is to be used in tests, but which might contain unreachable URLs
+		if workspaceShard.Annotations["experimental.core.kcp.io/unschedulable"] == "true" {
```

Comment:
unschedulable does not mean disabled. Does this code here mean that the syncer will stop work for those shards?

Reply:
I removed the commit.